### ISSUE TITLE
Deprecation warning for set-env

### ISSUE THREAD

**Issue by ZebraFlesh** [hashicorp/setup-terraform#55]
This action is now generating the following two deprecation notices:
```
Warning: The `set-env` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
Warning: The `add-path` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

Relevant portion of workflow yaml:

```yaml
env:
  TERRAFORM_VERSION: 0.13.4

jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - uses: hashicorp/setup-terraform@v1
      with:
        terraform_version: ${{ env.TERRAFORM_VERSION }}
```

**Comment by stijndehaes** [comment]
By updating the actions core package to the latest version this will be fixed automatically.
There is an automatic PR: https://github.com/hashicorp/setup-terraform/pull/52

But it fails because index.js needs to be updated.

**Comment by stijndehaes** [comment]
I have made a PR that fixes it: https://github.com/hashicorp/setup-terraform/pull/56

**Comment by chenrui333** [comment]
gentle ping @sudomateo

**Comment by aeschright** [comment]
Thanks for the report! This is fixed now in #63
